controllers/notification/mailgun_notification_delivery_service.go

```diff
@@ -107,7 +103,7 @@ func (s *MailgunNotificationDeliveryService) Send(notification *toolchainv1alpha
 		body = notification.Spec.Content
 	}
 
-	if subject == "" && body == "" {
+	if subject == "" || body == "" {
```

Comment:
Changed this, because it made sense to return if either body or subject is empty.